### PR TITLE
Set correct data in `check_in`s

### LINKED ISSUE #2496: Checkins are dropped when lots of Celery child tasks are started and `max-tasks-per-child` is reached
### How do you use Sentry?

Sentry Saas (sentry.io)

### Version

1.33.0

### Steps to Reproduce

Set `max-tasks-per-child` in Celery to 10000. Use `redis` as broker. Create a Celery Task that starts 12000 child tasks using `.delay()`. 
When `max-tasks-per-child` is reached our redis integration adds a lot of big breadcrumbs to the scope which are applied to all the success/failure checkins of the child spans. 
Because of the size of the breadcrumbs the checkin envelopes are too big and are dropped by the server. Leading to all child spans timing out because the success/failure checking never reaches sentry.

### Expected Result

All checkins are sent successfully

### Actual Result

The checkins are dropped because of size restriction on the server.

### PR DESCRIPTION
**Pull request by antonpirker**
Make sure that breadcrumbs are not in check-ins.

Fixes #2496

### CODE REVIEW

File: tests/test_crons.py [hunk 1]
```diff
@@ -220,3 +222,62 @@ def test_capture_checkin_sdk_not_initialized():
...
+        "duration",
+        "monitor_config",
+        "contexts",
+        # TODO: These fields need to be checked if valid for checkin:
+        "level",
+        "tags",
+        "extra",
+        "modules",
+        "server_name",
+        "sdk",
```
Comment (cleptric) [inline tests/test_crons.py:254]: This relates to my comment above, we should remove them.

File: tests/test_crons.py [hunk 2]
```diff
@@ -220,3 +222,62 @@ def test_capture_checkin_sdk_not_initialized():
...
+def test_scope_data_in_checkin(sentry_init, capture_envelopes):
+    sentry_init()
+    envelopes = capture_envelopes()
+
+    valid_keys = [
+        # Mandatory event keys
+        "type",
+        "event_id",
+        "timestamp",
+        "platform",
```
Comment (cleptric) [inline tests/test_crons.py:236]: This data is already sent in the envelope header, so we can remove it here as well.

Comment (antonpirker) [reply tests/test_crons.py:236]: we will do this in a separate pr

File: sentry_sdk/scope.py [hunk 3]
```diff
@@ -633,14 +626,57 @@ def _drop(cause, ty):
...
+        hint["attachments"] = attachments_to_send
+
+        if not is_check_in:
+            self._apply_level_to_event(event, hint, options)
+            self._apply_fingerprint_to_event(event, hint, options)
+            self._apply_user_to_event(event, hint, options)
+            self._apply_transaction_name_to_event(event, hint, options)
+            self._apply_transaction_info_to_event(event, hint, options)
+            self._apply_tags_to_event(event, hint, options)
+            self._apply_contexts_to_event(event, hint, options)
```
Comment (sentrivana) [inline sentry_sdk/scope.py:658]: Does this mean we never add the trace context to checkins? If [this](https://develop.sentry.dev/sdk/check-ins/) is up to date, the trace context should still be part of the checkin payload to link it with errors.

**Review by sentrivana — APPROVED** [review]
See one comment, else LGTM!

File: sentry_sdk/scope.py (hunk 3 above)
Comment (antonpirker) [reply sentry_sdk/scope.py:658]: oh good catch. will fix this

**Pull request merged**